### PR TITLE
fix(template): adapting to the new way resizing of the field plugin is handled

### PR DESCRIPTION
**Pull request by BibiSebi**
## What?
Resizing of the field plugin UI inside the iFrame has changed and been release in the previous sprint. 
Now once the plugin is loaded it sends a property to the Storyblok App --> `modalFullHeight: true`.
This property signals to the Container that in case the modal is open, the iframe should add `size=auto` inside the inline styles.

By adding this option, it should be easier for the user to create sticky headers and footers. Before there have been issues where such designs were not easily possible. This PR adds some styles to enable this feature. 

In the following video, I demonstrate the new possible implementation. The div with green background is sticky to top. The red div is sticky to bottom. 

https://www.loom.com/share/5cc5211c2ec6491c887ec1b78ad82336

## Why?

[JIRA: EXT-1926](https://storyblok.atlassian.net/browse/EXT-1926)

## How to test? (optional)
As the new functionality where the field plugin sends the new modalFullHeight property to the container, you need to run it locally and connect the local version of the field plugin to your template.

1. Rebuild field-plugin package --> cd packages/field-plugin && yarn build
2. Replacing dependency inside the react template to "@storyblok/field-plugin": "workspace:*"
3. cd into React Template and run --> yarn install && yarn dev

Then open the Partner Portal and go to the Field Plugin Development Environment.

### CODE REVIEW

**Comment by eunjae-lee** [comment]
> Can someone test the react template for now so that we know if it fixes all the issues. Once we approve it i can put it inside the other templates :)
> 
> PS: With this change there are no changes needed inside the storyfront!

it's working well!

**Review by eunjae-lee — APPROVED** [review]

File: packages/cli/templates/vue2/src/style.css [hunk 1]
```diff
@@ -1,7 +1,8 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap');
 
 html {
-  overflow: hidden;
+  overflow: auto;
+  hegiht: 100%;
```
Comment (demetriusfeijoo) [inline packages/cli/templates/vue2/src/style.css:5]:
```suggestion
  height: 100%;
```

some typo here @BibiSebi

**Review by demetriusfeijoo — COMMENTED** [review]
Are we going to also add the same adjustment for the vanilla JS version @BibiSebi?

**Comment by BibiSebi** [comment]
> Are we going to also add the same adjustment for the vanilla JS version @BibiSebi?
> 
> I've tested the adjustment made and it worked nicely. 💯
> 
> full-height-1 [ full-height-1 ](https://user-images.githubusercontent.com/1240591/269434071-4a5a7fd8-f8d0-434e-9631-4ca30dc3d9d9.gif) [ ](https://user-images.githubusercontent.com/1240591/269434071-4a5a7fd8-f8d0-434e-9631-4ca30dc3d9d9.gif)
> 
> full-height-2 [ full-height-2 ](https://user-images.githubusercontent.com/1240591/269434108-d07fb1de-7638-45d6-af52-388f06926f9e.gif) [ ](https://user-images.githubusercontent.com/1240591/269434108-d07fb1de-7638-45d6-af52-388f06926f9e.gif)

@demetriusfeijoo I added the styles also to the js template now :) and thank you for noticing the typo. My IDE is so weird recently, it never does what I want it to do :D

**Comment by eunjae-lee** [comment]
> > Are we going to also add the same adjustment for the vanilla JS version @BibiSebi?
> > I've tested the adjustment made and it worked nicely. 💯
> 
> @demetriusfeijoo I added the styles also to the js template now :) and thank you for noticing the typo. My IDE is so weird recently, it never does what I want it to do :D

time to try vim? 🤣

**Review by eunjae-lee — APPROVED** [review]

**Review by demetriusfeijoo — APPROVED** [review]

**Pull request merged**